app/views/install/codespaces/create-a-codespace.html

```diff
@@ -0,0 +1,87 @@
+{% set installEnvironment = "Codespaces" %}
+{% set current_step = '1' %}
+
+{% block pageTitle %}
+  Create a codespace - NHS prototype kit
+{% endblock %}
+
+{% extends 'install/codespace.html' %}
+
+{% block beforeContent %}
+  {{ breadcrumb({
+    items: [
+      {
+        href: "/",
+        text: "Home"
+      },
+      {
+    href: "/install/codespaces/",
+    text: "Install the kit in your web browser"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block stepHeader %}
+  Create a Codespace
+{% endblock %}
+
+{% block page_content %}
+
+  <p class="nhsuk-lede-text">To use the kit within a Codespace you will first need to create one.</p>
+
+  <h2>1. Go to the prototype repository</h2>
+
+  <p>Go to the GitHub repository for the prototype you wish to use. For this guide we will be starting a new prototype, using the
+    <a href="https://github.com/nhsuk/nhsuk-prototype-kit">NHS prototype kit repository</a>.
+  </p>
+
+  <p>If you using an existing prototype, go to step 2.</p>
+
+  <p>If you are starting a new prototype: copy the prototype kit files to your GitHub organisation. You can do this by 'forking' the repository.</p>
+  
+  <p><img class="app-img-guide nhsuk-u-margin-bottom-0" alt="Location of the fork button on the repository page" src="/images/codespaces/fork-button.png"></p>
```

Comment:
I wonder if we should avoid recommend using the 'fork' feature as that might cause some unexpected issues - eg if it defaults pull requests to being against the upstream repo instead of the forked one?

That said, the alternative right now would be to download the zip file and re-upload it which isn't ideal either.

Maybe it’s time to start looking at setting up a template repo sooner rather than later? 🤔

Reply:
Hmm yes you're right, forking is not right, didn't realise how it maintains a connection to the main repo. 

But following the normal way of downloading a zip to you computer then linking that up with a git repo then gets us back to doing stuff on your local machine… 

Don't know about templates but sounds like it could be what we're looking for.

Reply:
I'll change it to reference the download and re-upload process for now, even if that’s a bit of a pain. It should still be helpful for people who can install Visual Studio or GitHub Desktop, but can't get Node.js running.

Will spin out a separate issue to investigate template repos.